Files/UserControls/NavigationToolbar.xaml

```diff
@@ -1067,6 +1086,9 @@
                                         IsChecked="{x:Bind LayoutModeInformation, Converter={StaticResource LayoutModeConverter}, Mode=OneWay, ConverterParameter=DetailsInfo}"
                                         Style="{StaticResource ToggleButtonRevealStyle}"
                                         ToolTipService.ToolTip="Details View">
+                                        <RadioButton.KeyboardAccelerators>
+                                            <KeyboardAccelerator Key="Number2" Modifiers="Control,Shift" IsEnabled="{x:Bind AreKeyboardAcceleratorsEnabled, Mode=OneWay}" />
+                                        </RadioButton.KeyboardAccelerators>
```

Comment:
Can you move these shortcuts to ModernShellPage? When they are in the toolbar, they don't always work.

Reply:
Ok, I do it tomorrow.